projects/templates/plopfile.js

```diff
@@ -51,12 +52,25 @@ module.exports = function (plop) {
             {
                 type: 'input',
                 name: 'name',
-                message: 'SWC package name please',
+                message: 'SWC package name (i.e. color-area)',
```

Comment:
Does this require any changes to https://github.com/adobe/spectrum-web-components/blob/docs-readme-updates/documentation/guides/generating-components.md?

Reply:
Just pushed that update now